code/modules/loadout/categories/glasses.dm

```diff
@@ -0,0 +1,57 @@
+// --- Loadout item datums for glasses ---
+
+/// Glasses Slot Items (Moves overrided items to backpack)
+/datum/loadout_category/glasses
+	category_name = "Glasses"
+	category_ui_icon = FA_ICON_GLASSES
+	type_to_generate = /datum/loadout_item/glasses
+	tab_order = 2
```

Comment:
These should either be defines in one central place or even better, `/datum/loadout_category/head::tab_order + 1`